deploy-manage/deploy/cloud-enterprise/ece-configure-templates-index-management.md

```diff
@@ -20,8 +20,11 @@ In a template that creates a hot-warm architecture, you can use index curation t
 
 Configuring index management is part of the larger task of [creating deployment templates](ece-configuring-ece-create-templates.md) or editing them. The choices you make here determine which index management methods are available to your users when they create deployments.
 
-You should configure all index management methods that you want your users to be able to choose from when they create their deployments from your template. You can configure index curation, index lifecycle management, or both. If you are not sure which index management methods to configure, take a look at how your users are asked to [configure index management](https://www.elastic.co/guide/en/cloud-enterprise/current/ece-configure-index-management.html) when they customize their deployments.
+You should configure all index management methods that you want your users to be able to choose from when they create their deployments from your template. You can configure index curation, index lifecycle management, or both.
 
+::::{note}
+Index curation has been deprecated in favor of [index lifecycle management](/manage-data/lifecycle/index-lifecycle-management.md). For {{stack}} version 6.7 and later, any deployments using index curation will be prompted to [migrate to ILM](/manage-data/lifecycle/index-lifecycle-management/migrate-index-management.md).
+::::
```

Comment:
The removed paragraph didn't have much sense, more over when index curation is deprecated and we should recommend against it.

I've added the note that can be felt a bit repetitive because in the beginning of the doc we already mention that index curation is deprecated (i think it's beneficial also to remind it just before describing both options).